.go-version

```diff
@@ -1 +1 @@
-1.16.2
+1.18.2
```

Comment:
Why is upgrading Go version + dependencies part of these PRs?
Are these required to add "DENY_ALL" policy?   if not they should be separate PRs.

PRs should be as small as possible and limited to "single" change.